src/components/verification-grid-tile/verification-grid-tile.ts

```diff
@@ -50,10 +51,16 @@ export class VerificationGridTileComponent extends AbstractComponent(LitElement)
 
   public static readonly selectedEventName = "selected" as const;
 
+  // I use type: Object here so that the lit-plugin VSCode extension will throw
+  // an error if we use the model in an attribute without dot notation
   @provide({ context: gridTileContext })
-  @property({ attribute: false })
+  @property({ attribute: false, type: Object })
```

Comment:
![image](https://github.com/user-attachments/assets/ceeaacb1-8f95-44c2-8806-03fadbba96f7)

Why you don't want to use [model]="${this.model}"

https://github.com/runem/lit-analyzer/blob/master/docs/readme/rules.md#%EF%B8%8F-no-complex-attribute-binding

Reply:
Ok then you've probably taken the wrong lesson from that analyzer. It is saying, correctly, that binding complex object through attributes is not wise (without a converter) because attributes are all string serialized values.

If we don't use the property decorator here then we won't have an issue